GEOSldas_App/util/postproc/ObsFcstAna_stats/helper/write_nc4.py

```diff
@@ -0,0 +1,36 @@
+from netCDF4 import Dataset
+import numpy as np
+
+def write_sums_nc4(file_path, N_data, data_sum, data2_sum, oxf_sum, oxa_sum, fxa_sum, obs_param):
```

Comment:
I suggest renaming the file to "write_sums_nc4.py" for consistency with the py function contained in the file.  (When renaming the file, use "git mv old name new name" so that git can keep track of the renaming.)